tests/conftest.py

```diff
@@ -14,6 +14,26 @@
 from testfm.log import logger
 
 
+@pytest.fixture(scope='function')
+def setup_yum_exclude(request, ansible_module):
+    """This fixture is used for adding and then removing yum excludes in /etc/yum.conf file.
+    """
+    exclude = 'exclude=cat* bear*'
```

Comment:
It will be good to make such fixtures more generic. Can we pass this "exclude = 'exclude=cat* bear*'" as a variable to fixture?

Reply:
will check if we can do it.